cfn-resources/search-indexes/README.md

```diff
@@ -0,0 +1,13 @@
+# Mongodb::Atlas::SearchIndex
+
+Congratulations on starting development!
+
+Next steps:
+
+1. Populate the JSON schema describing your resource, `mongodb-atlas-searchindex.json`
+2. The RPDK will automatically generate the correct resource model from the
+   schema whenever the project is built via Make.
+   You can also do this manually with the following command: `cfn-cli generate`
+3. Implement your resource handlers by adding code to provision your resources in your resource handler's methods.
+
```

Comment:
Missing Example Section (with link) and Reference to Open API Docs (with link). See Clusters for good README file you can emulate: https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/cfn-resources/cluster